api/Sources/Api/Extend/Tagged.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 import PairQL
 import Tagged
 
-extension Tagged: PairInput where RawValue == UUID {}
+extension Tagged: @retroactive PairInput where RawValue == UUID {}
```

Comment:
all these `@retroactive` conformances are ok because we own one of the types, even if the swift compiler doesn't know that. there are upcoming changes to the language to express this without the override, at least for the concept of "packages", which may not help us, but they've acknowledged that what we're doing here is highly desirable, to conform some of our own types to protocols outside their originating package, so i'm ok with using this attribute pretty liberally.

see: https://forums.swift.org/t/amendment-se-0364-allow-same-package-conformances/71877/4